authentication/api/main.py

```diff
@@ -46,11 +50,11 @@ def get_authentication_credentials(
         )
     current_username_bytes = credentials.username.encode("utf8")
     is_correct_username = secrets.compare_digest(
-        current_username_bytes, conf.CLIENT_ID.encode("utf8")
+        current_username_bytes, conf.OAUTH_CLIENT_ID.encode("utf8")
```

Comment:
There's no reason to have a separate CLIENT_ID value, as we're only doing the simple check of "is the unencoded value the same as the value in our config?".